addon/components/freestyle-md-text.js

```diff
@@ -1,15 +0,0 @@
-import Ember from 'ember';
-import MDTextComponent from 'ember-remarkable/components/md-text';
-
-const { computed } = Ember;
-
-export default MDTextComponent.extend({
```

Comment:
Would you consider this component public API? We could consider leaving a stub version of the component that throws an error instructing the user what to use instead.